Resources/doc/admin/general-params.md

```diff
@@ -0,0 +1,180 @@
+# General builder params
+
+[go back to Table of contents][back-to-index]
+
+-----
+
+The default params for a new generator are as follows:
+```yaml
+generator: {{ generator }}
+params:
+    model: {{ namespace }}\{{ model_folder }}\{{ model_name }}
+    namespace_prefix: {{ namespace_prefix }}
+    bundle_name: {{ bundle_name }}
+    concurrency_lock: ~
+    i18n_catalog: Admin
+    credentials: ~
+    pk_requirement: ~
+    custom_blocks: ~
+    fields: ~
+    object_actions:
+        delete: ~
+    batch_actions:
+        delete: ~
+    actions: ~
+```
+
+They can be customized on a per admin basis, according to your needs.
+
+If you want to see the actual list/excel/new/edit builder documentation, check [this documentation][builders-doc].
+
+### Generator
+`generator` __default__: `admingenerator.generator.{{generator}}` __type__: `string`
+
+The generator is the most important part of the general config. It indicates the service that needs to be used for generator this particular admin page. By default, is is the generator service provided by this bundle for the model manager entered during admin creation.
+
+### Model
+`model` __default__: `{{ namespace }}\{{ model_folder }}\{{ model_name }}` __type__: `string`
+
+The model that this admin manages. Use the fully qualified model name.
+
+### Bundle
+`namespace_prefix` __default__: `{{ namespace_prefix }}` __type__: `string`
+`bundle_name` __default__: `{{ bundle_name }}` __type__: `string`
+
+The `namespace_prefix` and `bundle_name` together form the fully qualified bundle name. For example, in the case of the bundle `Acme/AdminBundle`, the `namespace_prefix` would be `Acme` and the `bundle_name` would be `AdminBundle`.
+
+### Concurrency lock
+`concurrency_lock` __default__: `false` __type__: `boolean`
+
+To protect your models edit action against concurrent modifications during long-running business transactions this parameter can be used. If enabled, before updating actually updating the object, this bundle will check if there were any modifications by comparing object versions. Simply set it to true and make sure that your model contains a version field.
+
+##### Versionable field for Doctrine ORM/ODM
+
+Add the `version` property to your model, with the correct annotation.
+
+```php
+class Article
+{
+    /**
+     * @ORM\Version()
+     * @ORM\Column(type="integer")
+     */
+    private $version;
+
+    public function getVersion(){
```

Comment:
It's part of [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#11-example) which is used by symfony2 core